.drone.starlark

```diff
@@ -601,6 +601,418 @@ config = {
+				'apiTags': 'apiTags-enc-UK',
+				'apiTrashbin': 'apiTrashbin-enc-UK',
+				'apiVersions': 'apiVersions-enc-UK',
+				'apiWebdavLocks': 'apiWebdavLocks-enc-UK',
+				'apiWebdavLocks2': 'apiWebdavLocks2-enc-UK',
+				'apiWebdavMove': 'apiWebdavMove-enc-UK',
+				'apiWebdavOperations': 'apiWebdavOperations-enc-UK',
+				'apiWebdavProperties': 'apiWebdavProperties-enc-UK',
+				'apiWebdavUpload': 'apiWebdavUpload-enc-UK',
+			},
+			'databases': [
+				'mysql:5.7',
+			],
+			'servers': [
+				'latest',
+			],
+			'phpVersions': [
+				'7.1',
+			],
+			'runCoreTests': True,
+			'cron': 'nightly',
+			'extraApps': {
+				'encryption': ''
+			},
+			'extraSetup': [
+				{
+					'name': 'configure-encryption',
+					'image': 'owncloudci/php:7.1',
+					'pull': 'always',
+					'commands': [
+						'cd /var/www/owncloud/server',
+						'php occ encryption:enable',
+						'php occ encryption:select-encryption-type user-keys --yes',
+						'php occ config:list'
+					]
+				}
+			],
+		},
+		'core-api-federation-encryption-userkeys-nightly': {
+			'suites': {
+				'apiFederation': 'Federation-enc-UK'
+			},
+			'databases': [
+				'mysql:5.7',
+			],
+			'servers': [
+				'latest',
+			],
+			'phpVersions': [
+				'7.1',
+			],
+			'runCoreTests': True,
+			'federatedServerNeeded': True,
+			'cron': 'nightly',
+			'extraApps': {
+				'encryption': ''
+			},
+			'extraSetup': [
+				{
+					'name': 'configure-app',
+					'image': 'owncloudci/php:7.0',
+					'pull': 'always',
+					'commands': [
+						'cd /var/www/owncloud/server',
+						'bash ./apps/user_ldap/tests/acceptance/setConfig.sh',
+						'php occ ldap:show-config',
+						'php occ ldap:test-config "LDAPTestId"',
+						'php occ user:sync "OCA\User_LDAP\User_Proxy" -m remove',
+						'php occ user:list',
+					]
+				},
+				{
+					'name': 'configure-app-on-federated-server',
+					'image': 'owncloudci/php:7.0',
+					'pull': 'always',
+					'commands': [
+						'cd /var/www/owncloud/federated',
+						'php occ market:install user_ldap',
+						'bash /var/www/owncloud/server/apps/user_ldap/tests/acceptance/setConfig.sh',
+						'php occ ldap:show-config',
+						'php occ ldap:test-config "LDAPTestId"',
+						'php occ user:sync "OCA\User_LDAP\User_Proxy" -m remove',
+						'php occ user:list',
+					]
+				},
+				{
+					'name': 'configure-encryption',
+					'image': 'owncloudci/php:7.1',
+					'pull': 'always',
+					'commands': [
+						'cd /var/www/owncloud/server',
+						'php occ encryption:enable',
+						'php occ encryption:select-encryption-type user-keys --yes',
+						'php occ config:list'
+					]
+				}
+			],
+		},
+		'core-cli-acceptance-encryption-userkeys-nightly': {
+			'suites': {
+				'cliTrashbin': 'cliTrashbin-enc-UK'
+			},
+			'databases': [
+				'mysql:5.7',
+			],
+			'servers': [
+				'latest',
+			],
+			'phpVersions': [
+				'7.1',
+			],
+			'runCoreTests': True,
+			'cron': 'nightly',
+			'extraApps': {
+				'encryption': ''
+			},
+			'extraSetup': [
```

Comment:
IMO this will also need the `configure-app` `extraSetup` block like in `core-api-federation-encryption-userkeys-nightly` because it will not get added from the `extraSetup` in `defaults`